Apps/WebClient/src/ClientApp/src/constants/entryType.ts

```diff
@@ -18,7 +30,7 @@ export class EntryTypeDetails {
     name!: string;
     description!: string;
     icon!: string;
-    component!: string;
+    component!: Component;
```

Comment:
Nice way of dealing with this, will clean up any imports in the future 👍

Reply:
It also resolves an issue when these were being used.  The composition API SFCs don't seem to automatically register components referenced dynamically as strings, so they would have needed to be registered explicitly, by adding a separate `<script>` tag and exporting an object that lists all of the components.  It looked a bit messy.